package.json

```diff
@@ -43,19 +43,20 @@
     "jest": "^26.6.3",
     "jsonwebtoken": "^8.5.1",
     "knex": "^0.21.6",
+    "module-alias": "^2.2.2",
```

Comment:
Let's get rid of this package if it's not in use. Remember to re-run `npm i` and commit the latest `package-lock.json` also